css-layout-classes/index.html

```diff
@@ -0,0 +1,88 @@
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1>Choose your character!</h1>
+      <div class="row">
+        <div class="columnThird">
+          <img src="./images/link.png" alt="" />
+        </div>
+        <div class="columnThird">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>
+            Super Smash Bros. Ultimate received universal acclaim from both
+            critics and players, with some critics calling it the best
+            installment in the series. It received praise for its large amount
+            of content and fine-tuning of existing Smash gameplay elements,
+            although its online mode was widely criticized.
+          </p>
+        </div>
+        <div class="columnThird">
+          <img src="./images/yoshi.png" alt="" />
+        </div>
+      </div>
+      <div class="row">
+        <div class="columnQuarter">
+          <img src="./images/kirby.png" alt="" />
+        </div>
+        <div class="columnQuarter">
+          <img src="./images/fox.png" alt="" />
+        </div>
+        <div class="columnQuarter">
+          <img src="./images/pikachu.png" alt="" />
+        </div>
+        <div class="columnQuarter">
+          <img src="./images/ness.png" alt="" />
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="columnHalf">
+          <h2>Donkey Kong</h2>
+          <p>
+            Donkey Kong is a series of video games featuring the adventures of a
+            gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981.
+            The franchise consists mainly of two game genres, but also includes
+            additional spin-off titles of various genres.
+          </p>
+          <p>
+            The puzzle games of the first genre are mostly single-screen
+            platform/action puzzle types, featuring Donkey Kong as the opponent
+            in an industrial construction setting.
+          </p>
+        </div>
+        <div class="columnHalf">
+          <img src="./images/dk.png" alt="" />
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="columnHalf">
+          <img src="./images/mario.png" alt="" />
+        </div>
+
+        <div class="columnHalf">
+          <h2>Mario</h2>
+          <p>
+            Mario is a fictional character in the Mario video game franchise,
+            owned by Nintendo and created by Japanese video game designer
+            Shigeru Miyamoto. Acting as the company's mascot, as well as being
+            the eponymous protagonist of the series, Mario has appeared in over
+            200 video games since his creation.
+          </p>
+          <p>
+            Depicted as a short, pudgy, Italian plumber who resides in the
+            Mushroom Kingdom, his advanetures generally center upon rescuing
+            Princess Peach from the Koopa villain Bowser.
+          </p>
```

Comment:
Should these be `column-half`? In the reference image are they `half` or more/less than `half` for each piece?

Also I don't see a CSS ruleset for `columnHalf` so where was this getting styled from?